Sources/Widgets/Widgets3D/ResliceCursorWidget/behavior.js

```diff
@@ -83,36 +85,93 @@ export default function widgetBehavior(publicAPI, model) {
     const image = model.widgetState.getImage();
     const imageSpacing = image.getSpacing();
     const imageBounds = image.getBounds();
-    const boundingBox = vtkBoundingBox.newInstance({ bounds: imageBounds });
+    const bounds = vtkBoundingBox.newInstance({ bounds: imageBounds });
 
     // Get the normal use to translate the center along
-    const normal = calldata.pokedRenderer
+    const dirProj = calldata.pokedRenderer
       .getRenderWindow()
       .getRenderers()[0]
       .getActiveCamera()
       .getDirectionOfProjection();
 
-    // Direction of the projection if the inverse of what we want
-    const translationVector = vtkMath.multiplyScalar(normal, -1);
+    // Direction of the projection is the inverse of what we want
+    const normal = vtkMath.multiplyScalar(dirProj, -1);
+
+    const XPlaneNormal = model.widgetState.getXPlaneNormal();
+    const YPlaneNormal = model.widgetState.getYPlaneNormal();
+    const ZPlaneNormal = model.widgetState.getZPlaneNormal();
+
+    // Find in which view we are
+    let otherViews = ['X', 'Y'];
+    let otherNormals = [XPlaneNormal, YPlaneNormal];
+    if (vtkMath.dot(normal, XPlaneNormal) > 0.9) {
+      otherViews = ['Y', 'Z'];
+      otherNormals = [YPlaneNormal, ZPlaneNormal];
+    } else if (vtkMath.dot(normal, YPlaneNormal) > 0.9) {
+      otherViews = ['X', 'Z'];
+      otherNormals = [XPlaneNormal, ZPlaneNormal];
+    }
```

Comment:
I'm surprised you do not already have utility functions for that. 
Please add it as utility function next to the "GetAssociatedLine..."